src/DeepHelloWorld/Tests/Frontend/angular/controllers/SayHelloController.spec.js

```diff
@@ -5,6 +5,7 @@ import moduleName from '../../../../Frontend/js/app/angular/name';
 describe('Controllers', function() {
   let $controller;
   let $rootScope;
+  let controller = null;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).